DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: rhino
 Title: A Framework for Enterprise Shiny Applications
-Version: 1.1.1.9001
+Version: 1.1.1.9004
```

Comment:
Please always bump by 1. This will naturally create a merge conflict when another PR is merged before this one, which is good: just merge `main` or rebase onto it and fix the version number,

Reply:
Dev version should now be `1.1.1.9002`, but this will be similar to #355.

Reply:
Bumping dev once more to `1.1.1.9003` after merging #355.